NOTES.md

```diff
@@ -0,0 +1,35 @@
+# Outline
+
+## Middleware
+
+- Fetches EUR conversion hourly/yearly -- resets cache
+- API:
+  - '/api/currencies/:currencyCode' -- returns currency object with all conversions, adds result to cache
+  - additionally, every return should include a marker indicating how long it will be valid for
+
+## Extension
+
+- On user request, converts the displayed currencies to the preferred base currency
+
+### Content script
+
+- Responsible for parsing the currencies on the page and reporting to the background script
+- Updates the currency text on the page and/or adds a tooltip for each displayed currency
```

Comment:
To go along with the above, we could provide way for user to report an erroneous conversion.

Reply:
That would be a v2 feature I think